src/parser/core/Combatant.ts

```diff
@@ -513,6 +468,20 @@ class Combatant extends Entity {
 
   // endregion
 
+  // region Tier
+  get tierPieces() {
+    return [this.head, this.shoulder, this.chest, this.legs, this.hands];
+  }
+
+  has2Piece(setId: number) {
```

Comment:
Is there a real need for setId to be passed in?
We can get the specID of the player here couldn't here and then filter from that?

Reply:
passing setId is now optional, if not passed, the tier setId will be calculated by mapping spec to class to setId